concepticondata/conceptlists/Stoll-1884-259.tsv

```diff
@@ -0,0 +1,260 @@
+Stoll-1884-259-153	153	übermorgen	the day after tomorrow	1179	DAY AFTER TOMORROW
+Stoll-1884-259-154	154	Monat	month	1370	MONTH
+Stoll-1884-259-155	155	Jahr	year	1226	YEAR
+Stoll-1884-259-156	156	Wald	forest	420	FOREST
+Stoll-1884-259-157	157	Berg	mountain	639	MOUNTAIN
+Stoll-1884-259-158	158	Ebene	plain	652	PLAIN
+Stoll-1884-259-159	159	Silber	silver	759	SILVER
+Stoll-1884-259-160	160	Eisen	iron	621	IRON
+Stoll-1884-259-161	161	Sand	sand	671	SAND
+Stoll-1884-259-162	162	Stein	stone	857	STONE
+Stoll-1884-259-163	163	Koth	dirt	25	DIRT
+Stoll-1884-259-164	164	Meer	sea	1474	SEA
+Stoll-1884-259-165	165	See	lake	624	LAKE
+Stoll-1884-259-166	166	Fluss	river	666	RIVER
+Stoll-1884-259-167	167	Feuer	fire	221	FIRE
+Stoll-1884-259-168	168	Rauch	smoke	778	SMOKE (EXHAUST)
+Stoll-1884-259-169	169	Asche	ash	646	ASH
+Stoll-1884-259-170	170	Baum	tree	906	TREE
+Stoll-1884-259-171	171	Baumast	branch	1531	BRANCH
+Stoll-1884-259-172	172	Blume	flower	239	FLOWER
+Stoll-1884-259-173	173	Blatt	leaf	628	LEAF
+Stoll-1884-259-174	174	Wurzel	root	670	ROOT
+Stoll-1884-259-175	175	Dorn	thorn	124	THORN
+Stoll-1884-259-176	176	Frucht	fruit	1507	FRUIT
+Stoll-1884-259-177	177	Brennholz	firewood	10	FIREWOOD
+Stoll-1884-259-178	178	Kohle	coal	2658	COAL
+Stoll-1884-259-179	179	Banane	banana	868	BANANA
+Stoll-1884-259-180	180	weiss	white	1335	WHITE
+Stoll-1884-259-181	181	schwarz	black	163	BLACK
+Stoll-1884-259-182	182	grün	green	1425	GREEN
+Stoll-1884-259-183	183	gelb	yellow	1424	YELLOW
+Stoll-1884-259-184	184	roth	red	156	RED
+Stoll-1884-259-185	185	Aguacate	avocado		
+Stoll-1884-259-186	186	Fichte	spruce	545	SPRUCE
+Stoll-1884-259-187	187	Tabak	tobacco	974	TOBACCO
+Stoll-1884-259-188	188	Weg	path	2252	PATH
+Stoll-1884-259-189	189	heiss	hot	1286	HOT
+Stoll-1884-259-190	190	kalt	cold	1287	COLD
+Stoll-1884-259-191	191	hart	hard	1884	HARD
+Stoll-1884-259-192	192	weich	soft	1856	SOFT
+Stoll-1884-259-193	193	viel	many	1198	MANY
+Stoll-1884-259-194	194	wenig	little	2924	A LITTLE
+Stoll-1884-259-195	195	nichts	nothing	687	NOTHING
+Stoll-1884-259-196	196	gross	big	1202	BIG
+Stoll-1884-259-197	197	klein	small	1246	SMALL
+Stoll-1884-259-198	198	gut	good	1035	GOOD
+Stoll-1884-259-199	199	böse, schlecht	evil, bad		
+Stoll-1884-259-200	200	breit	wide	1243	WIDE
+Stoll-1884-259-201	201	schmal	narrow	1267	NARROW
+Stoll-1884-259-202	202	leicht	light	1052	LIGHT (WEIGHT)
+Stoll-1884-259-203	203	sauer	sour	1906	SOUR
+Stoll-1884-259-204	204	süss	sweet	717	SWEET
+Stoll-1884-259-205	205	schön	beautiful	1427	BEAUTIFUL
+Stoll-1884-259-206	206	hässlich	ugly	1548	UGLY
+Stoll-1884-259-207	207	reich	rich	712	RICH
+Stoll-1884-259-208	208	arm	poor	1674	POOR
+Stoll-1884-259-209	209	krank	sick	1847	SICK
+Stoll-1884-259-210	210	schwer	heavy	1210	HEAVY
+Stoll-1884-259-211	211	alt	old	1229	OLD
+Stoll-1884-259-212	212	jung	young	1207	YOUNG
+Stoll-1884-259-213	213	fern	far	1406	FAR
+Stoll-1884-259-214	214	nahe	near	1942	NEAR
+Stoll-1884-259-215	215	fett	fat	1279	FAT (OBESE)
```

Comment:
Not sure about this one. Is it really OBESE? Could also be 'fat in texture', i.e. of meat.

Reply:
I guess in this case it would be better to leave it unmapped, right?

Reply:
If we can't disambiguate it with the help of the original source, yes.